package.json

```diff
@@ -35,7 +35,7 @@
     "@material-ui/styles": "^4.11.3",
     "axios": "^0.21.0",
     "bible-reference-range": "^1.0.1",
-    "datatable-translatable": "1.0.15",
+    "datatable-translatable": "file:.yalc/datatable-translatable",
```

Comment:
I have updated the datatable-translatable version to 1.0.16. You can use it here